services/tenant-ui/frontend/src/store/verifierStore.ts

```diff
@@ -18,7 +21,13 @@ export const useVerifierStore = defineStore('verifier', () => {
     return fetchList('/tenant/v1/verifier/presentations/', presentations, error, loading);
   }
 
-  return { presentations, selectedPresentation, loading, error, listPresentations };
+  async function getPresentationDetails(verifier_presentation_id: string) {
+    selectedPresentation.value = null;
```

Comment:
shouldn't purge the selectedPresentation unless you are filling it with the new object (seems doubtful and risky since we don't know exactly what we are going to do with it, but it points to the item in the collection, not this item).

Reply:
Good catch.